Wikipedia/Code/Session.swift

```diff
@@ -169,7 +169,9 @@ import Foundation
         guard let request = request(with: url, method: method, bodyParameters: bodyParameters, bodyEncoding: bodyEncoding) else {
             return nil
         }
-        return jsonDictionaryTask(with: request, completionHandler: completionHandler)
+        let task = jsonDictionaryTask(with: request, completionHandler: completionHandler)
+        task.resume()
+        return task
```

Comment:
The original intent was to have the methods named `jsonDictionaryTask` just return tasks without calling `resume()` so they could be tracked before resuming in the caller. As it is here, the completion might be called before it's tracked in the `ReadingListsAPIController`, so the resume should be there after the `self.track`

Reply:
Wouldn't this also apply to `jsonDecodableTaskWithDecodableError`?

Reply:
Yup, good catch. I have that that wrong there as well